understanding/label-in-name.html

```diff
@@ -273,82 +190,61 @@ <h4>Mathematical expressions and formulae</h4>
                <section id="accessible-name-and-description-computation-specification">
                   
                   
-                  <h3>Accessible Name and Description Computation specification</h3>
+                  <h3>Accessible Name and Description Computation 仕様</h3>
                   
-                  <p>It is important to understand how the accessible name is derived. The <a href="https://www.w3.org/TR/accname-1.1/">Accessible Name and Description Computation 1.1</a> and the <a href="https://w3c.github.io/html-aam/#accessible-name-and-description-computation">HTML Accessibility API Mappings 1.0</a> describe how the accessible name is computed, including which attributes are considered
-                     in its calculation, and in what order of preference. If a component has multiple possible
-                     attribute values that could be used for its accessible name, only the most preferred
-                     of those values will be computed. None of the other, less preferred values will be
-                     part of the name. For the most part, existing established programmatic relationships
-                     between labels and controls are reinforced by the specification.
+                  <p>アクセシブルな名前 (accessible name) がどのように派生するかを理解することが重要である。<a href="https://www.w3.org/TR/accname-1.1/">Accessible Name and Description Computation 1.1</a> 及び <a href="https://w3c.github.io/html-aam/#accessible-name-and-description-computation">HTML Accessibility API Mappings 1.0</a> は、アクセシブルな名前 (accessible name) の計算方法を説明する。これには、計算で考慮される属性や優先順位などが含まれる。コンポーネントに、アクセシブルな名前 (accessible name) に使用できる複数の属性値がある場合、それらの値の中で最も優先されるものだけが計算される。他のあまり好ましくない値は名前 (name) の一部にはならない。ほとんどの場合、ラベルとコントロールの間に確立された既存のプログラム上の関係が仕様によって強化されている。
```

Comment:
less preferred values は「優先度がより低い」という意味かと……

```suggestion
                  <p>アクセシブルな名前 (accessible name) がどのように派生するかを理解することが重要である。<a href="https://www.w3.org/TR/accname-1.1/">Accessible Name and Description Computation 1.1</a> 及び <a href="https://w3c.github.io/html-aam/#accessible-name-and-description-computation">HTML Accessibility API Mappings 1.0</a> は、アクセシブルな名前 (accessible name) の計算方法を説明する。これには、計算で考慮される属性や優先順位などが含まれる。コンポーネントに、アクセシブルな名前 (accessible name) に使用できる複数の属性値がある場合、それらの値の中で最も優先されるものだけが計算される。優先度が低い他の値は名前 (name) の一部にはならない。ほとんどの場合、ラベルとコントロールの間に確立された既存のプログラム上の関係が仕様によって強化されている。
```